jobs/myStoriesInSprint.js

```diff
@@ -9,27 +9,59 @@ module.exports = {
     'jira'
   ],
   userInfoNeeded: [
-    'jiraUserId'
+    'jiraUserId',
+    'jiraTeam'
   ],
   hiddenFromHelp: false,
   phrases: [
     'what are my stories this sprint?',
-    'stories this sprint'
+    'stories this sprint',
+    'what are my stories'
   ],
   fn: function ({
                   bot,
                   message,
                   jira,
                   userInfo
                 }) {
+    let isTeam = false;
+    let isMine = false;
+    let assignee = `assignee = ${userInfo.jiraUserId}`;
+    let ownershipString = 'Your'
+    const messageText = message.text.toLowerCase();
+    if (messageText.includes('team')) {
+      isTeam = true}
+    if (messageText.includes('my') || messageText.includes('mine')) {
+      isMine = true}
+    if (isTeam && isMine) {
+      assignee = `assignee in membersOf(${ userInfo.jiraTeam })`;
+      ownershipString = 'Your team\'s';
+    }
+    else if (isTeam) {
+      const team = utils.getTeamFromMessageText(messageText);
+      if (!team) {
+        bot.reply(message, 'Sorry, I\'m not what team you\'re referring to. Type `@QA-Bot list teams` for the ones that I know.');
+        return };
+      assignee = `assignee in membersOf("${ team }")`;
+      ownershipString = `The "${ team }" team's`;
+    };
```

Comment:
a lot of the logic here is pretty much copy paste from p2p3Links and myOpenBugs, this should probably be a utils function